app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +120,50 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_exposed_ports(exposed_ports)
+        exposed_ports.to_h.keys.each_with_object({}) do |port, h|
+          n, p = port.to_s.split('/')
```

Comment:
@enoodle do we want to be safe on this one and specify the number of splits we expect? (1)